cartridges/int_bolt_embedded_sfra/cartridge/client/default/js/account.js

```diff
@@ -170,6 +170,14 @@ exports.setupListeners = async function () {
         }
     });
 
+    // eslint-disable-next-line no-undef
+    Bolt.getInstance().on('authorize_modal_closed', () => {
+        var disabledAttr = $('.submit-customer').attr('disabled');
+        if (typeof disabledAttr !== 'undefined' && disabledAttr !== false) {
```

Comment:
Based on this [article](https://www.freecodecamp.org/news/falsy-values-in-javascript) explaining Falsy values in JS. I think it's good to use `if(!disabledAttr)` as condition

Reply:
Make sense, just updated